Nfield.Quota.Tests/QuotaFrameValidatorTests.cs

```diff
@@ -290,6 +290,64 @@ public void Frame_HappyPath()
             Assert.That(result.IsValid, Is.True);
         }
 
+        [Test]
+        [TestCase(0, true)]
+        [TestCase(1, true)]
+        [TestCase(-1, false)]
+        public void Frame_MaxOvershootShouldBePositiveOnLevels(int overshoot, bool isValid)
+        {
+            var quotaFrame = new QuotaFrameBuilder()
+                .VariableDefinition("var1", new[] { "a", "b" })
+                .Structure(f => f.Variable("var1"))
+                .Build();
+
+            var level = quotaFrame["var1", "a"];
+            level.MaxOvershoot = overshoot;
+
+            var validator = new QuotaFrameValidator();
+            var result = validator.Validate(quotaFrame);
+
+            Assert.That(result.IsValid, Is.EqualTo(isValid));
+
+            if (!isValid)
+            {
+                Assert.That(result.Errors.Single().ErrorMessage,
+                    Is.EqualTo($"Max overshoot '{overshoot}' is invalid for frame level id '{level.Id}' with name '{level.Name}'. Max overshoot values can only be set on leaf nodes, and must be positive."));
```

Comment:
maybe consider checking with "contains" instead of "equal" on error messages

Reply:
All other tests in this file use `Equals`, and I think it's actually quite important that the error messages here are good, because they are part of the public API as well. Besides that, a lot of the messages generated by the validator are very similar, so you would end up checking most of the message either way.

I can change the word `EqualTo` to `Contains`, but I don't see much value in it. If you think it's important, I will do it :smile:

Reply:
I usually check with contains in case there are line breaks or characters that might end up being different on different machines because of encoding and whatnot. No need to change it